pages/vi/pi-setup.md

```diff
@@ -36,7 +36,7 @@ _**NOTE:**_ The latest image will be at the bottom of the download page and will
     * Click "Flash!".  The process will take approximately 10 minutes to complete.
 1. Once it's done burning, you can view the contents in the `boot` partition of your microSD card using a file explorer.
 1. Unmount and safely eject the microSD card from your computer. 
-
+**NOTE:** If you run into any issues with flashing, to try and update Etcher.
```

Comment:
Please make the NOTE formatting consistent with the other notes (ie. blue box).

Reply:
@kjong Thank you for your advice. I will be more careful.